### ISSUE TITLE
Two types of tin ore

### ISSUE THREAD

**Issue by AeroyrHwrdd** [Zeno410/UndergroundBiomesConstructs1.7#12]
Hey, I have two types of tin ore in UBC, only one is conforming to Soartex texture and it's not the one being used. Is this related to Tinker's Construct tin? Any idea about getting to change the texture. I have a tin_overlay already there.

**Comment by Zeno410** [comment]
Each mod that provides an ore produces a different block with a different texture. Tinker's has it's own, which IIRC is a mix of light brown and red dots. You'll probably have to turn off tin from Tinker's and add it from some other mod.

Another possibility is that tinker's uses a different name. You could add the texture to your pack under that name as well.

**Comment by AeroyrHwrdd** [comment]
That's right, red and light brown. Cheers. Just trying to get my head around where things pull the textures from when its a different mod because it has a UBC compatibility, so it's overlaying on all the UBC stone types just can't see the texture being pulled in the UBC mod textures folder. Trying to understand how the code works. Also had a problem with Immersive Engineering Bauxite spawning. I didn't see it spawning even with just a normal stone.  

UPDATE: I have it. Tinker's textures have ore_tin_overlay.png. It wasn't yet textured by Soartex. I has done it now though. All working.
